tests/ImageSharp.Tests/Formats/Tiff/TiffDecoderTests.cs

```diff
@@ -53,6 +53,27 @@ public void Identify(string imagePath, int expectedPixelSize, int expectedWidth,
             }
         }
 
+        [Theory]
+        [InlineData(TestImages.Tiff.RgbLzw_NoPredictor_Multistrip, TiffByteOrder.LittleEndian)]
+        [InlineData(TestImages.Tiff.RgbLzw_NoPredictor_Multistrip_Motorola, TiffByteOrder.BigEndian)]
+        public void ByteOrder(string imagePath, TiffByteOrder expectedByteOrder)
+        {
+            var testFile = TestFile.Create(imagePath);
+            using (var stream = new MemoryStream(testFile.Bytes, false))
+            {
+                IImageInfo info = Image.Identify(stream);
+
+                Assert.NotNull(info.Metadata);
+                Assert.Equal(expectedByteOrder, info.Metadata.GetTiffMetadata().ByteOrder);
+
+                // todo: it's not a mistake?
+                stream.Seek(0, SeekOrigin.Begin);
```

Comment:
@JimBobSquarePants position zeroing must be done manually, is that correct?

Reply:
The manual positioning needs to happen because `Identify` has already read the input stream.